src/models/EventMessage.ts

```diff
@@ -54,20 +54,18 @@ export interface EventMessage {
 	parsedMessages: ParsedMessage[] | null;
 }
 
-export interface EventMessageItem extends EventMessage {
-	parsedMessage: ParsedMessage | null;
-}
-
 export interface ParsedMessage {
 	match: boolean;
 	id: string;
 	message: MessageBody;
 }
 
-export function isScreenshotMessage(message: EventMessageItem): boolean {
-	return message.parsedMessage
-		? /image\/\w+/gi.test(message.parsedMessage?.message.metadata.messageType)
-		: false;
+export function isScreenshotMessage(message: EventMessage): boolean {
+	return Boolean(
+		message.parsedMessages?.filter(parsedMessage =>
```

Comment:
so following this if some message has image than every parsedMessage is a screenshot message ?